File: src/main/java/com/p2p/lending/dao/ClapplyforDao.java
package com.p2p.lending.dao;

import java.util.List;
import java.util.Map;

import com.p2p.lending.entity.Clapplyfor;

public interface ClapplyforDao {
	
	public List<Clapplyfor> queryClapplyfors(Map<String, Object> map);
	
	public int insertClapplyfor(Map<String, Object> map);
	
	public int updateClapplyforState(Map<String, Object> map);

}

File: src/main/java/com/p2p/lending/dao/ClapplyforDao.kt
package com.p2p.lending.dao

import com.p2p.lending.entity.Clapplyfor

interface ClapplyforDao {

    fun queryClapplyfors(map: Map<String, Any>): List<Clapplyfor>

    fun insertClapplyfor(map: Map<String, Any>): Int

    fun updateClapplyforState(map: Map<String, Any>): Int

}
